include/seqan/basic/allocator_interface.h

```diff
@@ -204,6 +204,7 @@ allocate(T const &,
     if (data)
         SEQAN_PROADD(SEQAN_PROMEMORY, count * sizeof(TValue));
 #endif
+	memset((void*)data, 0, count * sizeof(TValue));
```

Comment:
Now after I changed it back to memset 0, razers and microrazers tests are failing on Clang, but they were ok yesterday with the same code. How can this be?